lib/tests/streamlit/web/cli_test.py

```diff
@@ -404,6 +404,38 @@ def test_activate_command(self):
             self.runner.invoke(cli, ["activate"])
             mock_credential.activate.assert_called()
 
+    @tempdir()
+    def test_email_send(self, temp_dir):
```

Comment:
These two tests I think should go in `credentials_test.py` (because they're testing the behavior of `Credentials` itself, rather than the CLI module that happens to _use_ Credentials).

Reply:
Whoops, didn't see that one existed -- will do